File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/uploads/provider/CourseUploadsProvider.java
package org.wikiedufoundation.wikiedudashboard.course_detail.uploads.provider;

import org.wikiedufoundation.wikiedudashboard.helper.PresenterCallback;

public interface CourseUploadsProvider {
    void requestCourseUploads(String url, PresenterCallback presenterCallback);
}

File: app/src/main/java/org/wikiedufoundation/wikiedudashboard/course_detail/uploads/provider/CourseUploadsProvider.kt
package org.wikiedufoundation.wikiedudashboard.course_detail.uploads.provider

import org.wikiedufoundation.wikiedudashboard.helper.PresenterCallback

interface CourseUploadsProvider {
    fun requestCourseUploads(url: String, presenterCallback: PresenterCallback<*>)
}
